### DISCUSSION TITLE
Backspace or Del key on selected block calls component:deselected event

### DISCUSSION THREAD

**Discussion by vladninja** [GrapesJS/grapesjs#4186] [Q&A]
Hi. When the component have model defaults: ```removable: false``` and ```selectable: true``` after the selecting a block and keypressing "Backspace" or "Delete" key the block is deselecting and ```component:deselected``` event fired.
Is there any chance to change this behavior (disable this functionality or some way to prevent this event)?

**Comment by artf** [comment]
Yeah, confirmed. The fix is ready for the next release.
